Neos.TimeableNodeVisibility/Classes/Command/TimeableNodeVisibilityCommandController.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Neos\TimeableNodeVisibility\Command;
+
+use Neos\Flow\Annotations as Flow;
+use Neos\Flow\Cli\CommandController;
+use Neos\ContentRepository\Core\SharedModel\Workspace\WorkspaceName;
+use Neos\ContentRepository\Core\Factory\ContentRepositoryId;
+use Neos\TimeableNodeVisibility\Service\TimeableNodeVisibilityService;
+use Neos\TimeableNodeVisibility\Domain\HandlingResult;
+
+#[Flow\Scope('singleton')]
+class TimeableNodeVisibilityCommandController extends CommandController
+{
+    #[Flow\Inject]
+    protected TimeableNodeVisibilityService $timeableNodeVisibilityService;
+
+    public function executeCommand(string $contentRepository = 'default', bool $quiet = false): void
```

Comment:
_maaaaaayyybee_ we should allow to pass multiple crIds so this command is more cost efficient than having to run nTimes

Reply:
Hmm no..., if you want that, you could implement your own command. And as each CR has it's own storage, there is not that much you could save here.